buttons/README.md

```diff
@@ -58,9 +60,39 @@ Parameters | Descriptions
 `ripple: RippleTheme = VitaminTheme.ripples.brand` | The ripple effect applied on the component
 `onClick: () -> Unit` | The callback to be called when the user click on the button
 
+
+**Icon Buttons**
+
+```kotlin
+object VitaminIconButtons {
+    @Composable
+    fun Primary(
+        modifier: Modifier = Modifier,
+        icon: Painter,
+        enabled: Boolean = true,
+        colors: ButtonColors = VitaminButtonsColors.primary(),
+        sizes: VitaminButtonSizes = VitaminButtonsSizes.medium(),
+        borders: ButtonBorders = VitaminButtonsBorders.none(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand,
+        onClick: () -> Unit
+    )
+}
+
+```
+Parameters | Descriptions
+-- | --
+`modifier: Modifier = Modifier` | The `Modifier` to be applied to the component
+`icon: Painter` | The icon to be displayed at the start or the end of the button container
```

Comment:
Not good to copy paste! :D

It is the documentation of the button with an icon.

Reply:
Think to update the kdoc too